packages/gestalt/src/Popover/InternalPopover.flowtest.js

```diff
@@ -0,0 +1,7 @@
+// @flow strict
```

Comment:
We don't really need this file. flowtest.js files are pretty boilerplater. we don't get much value in this case